source/js/teaser-ellipsis.js

```diff
@@ -29,7 +31,8 @@ var getWidth = function(textLine) {
  * @return {Integer} parent's width
  */
 var getParentsWidth = function(textLine) {
- 
+  if(textLine.innerText == "") {return 0;}
```

Comment:
I think it makes it more readable if we move the content of the if condition path to the next line. I think it is too easy to over read it here as the whole function continues.